packages/vue-components/src/panels/PanelBase.js

```diff
@@ -94,6 +94,9 @@ export default {
     hasSrc() {
       return this.src && this.src.length > 0;
     },
+    getSrcWithoutFragment() {
```

Comment:
let's touch up the naming (omit the `get`) per the conventions here https://vuejs.org/v2/guide/computed.html (you can have getters / setters for computed properties as well)